extensions/godslayerakp/ws.js

```diff
@@ -0,0 +1,283 @@
+// Name: WebSocket
+// Description: an extension for connecting to websocket servers like discord and cloudlink.
+// By: RedMan13 <https://scratch.mit.edu/users/RedMan13/>
+
+(function (Scratch) {
+  "use strict";
+  if (!Scratch.extensions.unsandboxed)
+    throw "can not load out side unsandboxed mode";
+
+  /* ------- BLOCKS -------- */
+  const { BlockType, Cast, ArgumentType } = Scratch;
+
+  class WebRequests {
+    /**
+     * no need to install runtime as it comes with Scratch var
+     */
+    constructor() {
+      this.instances = {};
+    }
+    getInfo() {
+      return {
+        id: "websocket",
```

Comment:
do you have a good reason why you want `websocket` instead of something based on your username like other extensions?

Reply:
> do you have a good reason why you want `websocket` instead of something based on your username like other extensions?

no... just didnt think about it

Reply:
ok, thanks, just wanted to make sure compatibility with another Scratch mod was not one of the goals